README.md

```diff
@@ -29,3 +29,117 @@ The installer scripts also work fine without docker; just run the following two
 
 The unpacking requires recent versions of msitools (0.98) and libgcab
 (1.2); sufficiently new versions are available in e.g. Ubuntu 19.04.
+
+
+# Build instructions for native environments
+
+The following instructions are for setting up MSVC without docker.
+
+## Prerequisites
+
+```bash
+apt-get update
+apt-get install -y wine64-development python msitools python-simplejson python-six ca-certificates winbind
+```
+
+## Installation
+
+We're going to install it into `~/my_msvc` to avoid needing root privileges on a non-contained system.
+You can change the location to wherever you want, even real `/opt/` if you want to deal with chown'ing all files to root before moving
```

Comment:
Please end sentences with a period.

Also currently, the installation path is hardcoded in the installed `msvcenv.sh` so one can't easily e.g. install into a directory under the home directory and then move them with sudo. So I'd suggest maybe leaving out the last sentence altogether (or clarify that the installation directory currently isn't movable from the path it was installed into).

Reply:
Oh! I didn't know! I reworded it to point this out.